website/docs/docs/cloud/manage-access/self-service-permissions.md

```diff
@@ -1,42 +1,81 @@
 
-## Read-Only vs. Developer License Types
+The group access permissions are as follows:
 
-Users configured with Read-Only license types will experience a restricted set of permissions in dbt Cloud. If a user is associated with a _Member_ permission set and a Read-Only seat license, then they will only have access to what a Read-Only seat allows. See [Seats and Users](/docs/cloud/manage-access/seats-and-users) for more information on the impact of licenses on these permissions.
+- **Owner** &mdash; Full access to account features.
+- **Member** &mdash; Robust access to the account with restrictions on features that can alter billing or security.
+- **Everyone** &mdash; A catch-all group for all users in the account. This group does not have any permission assignments beyond the user's profile. Users must be assigned to either the Member or Owner group to work in dbt Cloud. 
 
-## Owner and Member Groups in dbt Cloud Enterprise  
+## Licenses
 
-By default, new users are added to the Member and Owner groups when they onboard to a new dbt Cloud account. Member and Owner groups are included with every new dbt Cloud account because they provide access for administrators to add users and groups, and to apply permission sets.
+You assign licenses to every user onboarded into dbt Cloud. You only assign Developer-licensed users to the Owner and Member groups. The groups have no impact on Read-only or IT licensed users.
 
-You will need owner and member groups to help with account onboarding, but these groups can create confusion when initially setting up SSO and RBAC for dbt Cloud Enterprise accounts as described in the [Enterprise Permissions](enterprise-permissions) guide. Owner and Member groups are **account level** groups, so their permissions override any project-level permissions you wish to apply.
+There are three license types:
 
-After onboarding administrative users and configuring RBAC/SSO groups, we recommend the following steps for onboarding users to a dbt Cloud Enterprise account.
+- **Developer** &mdash; The default license. Developer licenses don't restrict access to any features, so users with this license should be assigned to either the Owner or Member group. You're allotted up to 8 developer licenses per account.
+- **Read-Only** &mdash; Read-only access to your project, including environments dbt Explorer. Doesn't have access to account settings at all. Functions the same regardless of group assignments. You're allotted up to 5 read-only licenses per account.
+- **IT** &mdash; Partial access to the account settings including users, integrations, billing, and API settings. Cannot create or edit connects or access the project at all. Functions the same regardless of group assignments.
 
+See [Seats and Users](/docs/cloud/manage-access/seats-and-users) for more information on the impact of licenses on these permissions.
 
-### Prerequisites
+## Table of groups, licenses, and permissions
 
```

Comment:
For this whole section, I wonder if it's maybe a bit simpler to:
- have an intro sentence before the key
- then repurpose the permissions bullets for each table so there's an intro section before each table
- make it clear in the first column that the fields are for UI fields in dbt Cloud (i struggled at first)

e.g.:
```
The following tables outline the access that users have if they are assigned a Developer license and the Owner or Member group, Read-only license, or IT license. Each role grants different levels of access to specific user interface (UI) fields within the Cloud UI, such as Connections, Credentials, and more.

Key:
(W)rite — Create new or modify existing. Includes send, create, delete, allocate, modify, and read.
(R)ead — Can view but can not create or change any fields.
No value — No access to the feature.

#### Account-level permissions
These permissions apply to UI fields that manage the overall dbt Cloud account, such as account settings, billing, and user invitations.
| UI field | Owner | Member | Read-online license | IT license |
| ------- | ------ | -------- | ------------------- | ---------- |
...text. 

```

Reply:
There is a project in October to completely overhaul this whole permissions tables thing. Will take it up with that project.